app/migrations/versions/afdbf2c8d633_.py

```diff
@@ -0,0 +1,29 @@
+"""empty message
```

Comment:
Nit: Migration message should be populated

Reply:
Hahaha whoops it's adctually on the line below lol!